fbw-a32nx/src/systems/fmgc/src/guidance/vnav/AtmosphericConditions.ts

```diff
@@ -1,5 +1,5 @@
-//  Copyright (c) 2022 FlyByWire Simulations
-//  SPDX-License-Identifier: GPL-3.0
+// Copyright (c) 2021-2023 FlyByWire Simulations
```

Comment:
Why is this being changed? Nothing is added to the file to merit a new claim in 2021 or 2023.